tests/System/rms_scripts.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+# This is a generic list of commands run from a client for testing/certification purposes.
+#
+# Submitter should follow through the logs
+
+echo " "
+echo " "
+echo " ########################## RMS #############################"
+echo " "
+echo " "
+
+echo "dirac-login dteam_user"
+dirac-login dteam_user
+if [[ "${?}" -ne 0 ]]; then
+   exit 1
+fi
+
+dirac_user=$( dirac-proxy-info | awk '/^username / {print $3}' )
+userdir="/dteam/user/$( echo "$dirac_user" | cut -c 1)/$dirac_user"
+
+echo "dirac-dms-put-and-register-request"
+dirac-dms-put-and-register-request $dirac_user$( date +%s ) $userdir/Dirac_Scripts_Test_Directory/resolv.conf /etc/resolv.conf RAL-SE
+if [[ "${?}" -ne 0 ]]; then
+   exit 1
+fi
+
+echo "## --> This should be treated rather fast, we anyway wait 60 seconds"
+sleep 60
```

Comment:
you should loop, looking at the status of `dirac-rms-request` output

Reply:
I know that would be more precise, but that would require to first parse the output of `dirac-dms-put-and-register-request` to know which request to look at with `dirac-rms-request`, and then parse the output of this last command. This would be rather fragile (parsing the output is not the most stable thing...).

Reply:
True, but that allows you to test the two directions: client -> server and server->client, which is important in particular for the changes you've recently introduced